internal/provider/data_source_policy_set.go

```diff
@@ -53,6 +53,18 @@ func dataSourceTFEPolicySet() *schema.Resource {
 				Optional:    true,
 			},
 
+			"agent_enabled": {
+				Description: "Whether the policy set is executed in the TFC agent. True by default for OPA policies",
+				Type:        schema.TypeBool,
+				Optional:    true,
+			},
+
+			"policy_tool_version": {
+				Description: "The policy tool version to run the policy evaluation against",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
```

Comment:
Since this is a data source these attributes should be marked as `Computed: True`. `Optional` or `Required` would indicate this attribute is an argument -- With respect to data sources the only attributes that should be arguments are attributes you can use to identify that particular resource.